doc/write_your_own/connections/filePickerSimple.ipynb

```diff
@@ -126,8 +108,8 @@
     "\n",
     "files = ipywidgets.SelectMultiple(\n",
     "    description=path.value,\n",
```

Comment:
This can get very scruffy if the file path is long. How about setting it to 'Dataset(s)'?